packages/core/src/components/Tabs/Tab.jsx

```diff
@@ -65,6 +84,12 @@ Tab.propTypes = {
    */
   onClick: PropTypes.func,
   /**
+   * Click event called with the following arguments:
+   * [`SyntheticEvent`](https://facebook.github.io/react/docs/events.html),
+   * `id`, `panelId`
```

Comment:
Just noticed I originally documented these arguments incorrectly (here and in the `onClick` documentation).

The order is incorrect and it's missing the `href` argument. This should actually be: `SyntheticEvent `, `panelId`, `id`, `href`